go/vt/sqlparser/ast.go

```diff
@@ -522,6 +529,16 @@ type (
 		Shards    string
 	}
 
+	// CreateProcedure represents a CREATE PROCEDURE statement.
+	CreateProcedure struct {
+		Name        TableName
+		Comments    *ParsedComments
+		IfNotExists bool
+		Definer     *Definer
+		Params      []*ProcParameter
+		Statement   CompoundStatement
```

Comment:
If you want to take the terminology from mysql doc
it is called `routine body` which is `simple statement such as SELECT or INSERT, or a compound statement written using BEGIN and END.`

Reply:
Done